cmd/storj-sim/network.go

```diff
@@ -260,6 +260,10 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 		consoleAuthToken := "secure_token"
 
+		satelliteDB := flags.SatelliteDB
+		if os.Getenv("STORJ_POSTGRES_TEST") != "" {
+			satelliteDB = os.Getenv("STORJ_POSTGRES_TEST")
```

Comment:
This is for different purpose and probably shouldn't override storj-sim behavior.

Reply:
Also for command line flags the usual behavior is that command line flags override environment variables not the other way around.